otel-extensions/src/main/java/org/hypertrace/agent/otel/extensions/HypertraceGlobalIgnoreMatcher.java

```diff
@@ -55,6 +81,16 @@ public Result classloader(ClassLoader classLoader) {
     if (name.startsWith("com.singularity.") || name.startsWith("com.yourkit.")) {
       return Result.IGNORE;
     }
+    if (CLASS_LOADER_GET_NAME != null) {
```

Comment:
Can you explain why testing against `name` does not work? 

What is the `name` when the `getName` returns `MultiTenantAgentClassLoader`

Reply:
Based on the log messages, I tried using the name, but I thought that the class loader wasn't namespaced in a cisco package so my first pass at this solution was adding the check `"MultiTenantAgentClassLoader".equals(name)` to the `if(..)` condition above. I was surprised when it didn't work, so I went with this one. Turns out, the class name is namedspaced properly and is `com.cisco.mtagent.entry.MTAgent$MultiTenantAgentClassLoader` so we definitely don't need all of this extra complexity. I'm very happy to simply  check for if the class loader class name starts with `com.cisco.` and being more comortable knowing this should be relatively stable over time 😄 Thanks for asking, much happier with the solution we landed on as a result!